tests/MongoDB.Driver.Tests/RetryableWritesTests.cs

```diff
@@ -43,6 +43,26 @@ public void Insert_with_RetryWrites_true_should_work_whether_retryable_writes_ar
             }
         }
 
+        [SkippableFact]
+        public void Retryable_write_operation_should_throw_custom_exception_on_servers_using_mmapv1()
+        {
+            RequireSupportForRetryableWrites();
+            RequireServer.Check().StorageEngine("mmapv1");
+
+            using (var client = GetClient(builder =>  { }))
+            using (var session = client.StartSession())
+            {
+                var database = client.GetDatabase(DriverTestConfiguration.DatabaseNamespace.DatabaseName);
+                var collection = database.GetCollection<BsonDocument>(DriverTestConfiguration.CollectionNamespace.CollectionName);
+                var document = new BsonDocument("x", 1);
+                var exception = Record.Exception(()=>collection.InsertOne(document));
```

Comment:
no spaces around `=>`.
it can be simply fix via `ctrl` + `k` + `d` (default vs hotkeys)

Reply:
Fixed